#!/usr/bin/env python
PACKAGE = "hanse_wallfollowing"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("desiredDistance", double_t, 0, "Desired distance to wall", 1.5, 0.1, 5.0)
gen.add("maxSpeed", double_t, 0, "Max. Speed", 0.7, 0.2, 1.0)
gen.add("echoSounderAngle", double_t, 0, "Winkel um den der Echosounder nach vorn gedreht ist (in Grad)", 26.57, 0.0, 45.0)

exit(gen.generate(PACKAGE, "wallfollowing", "WallFollowing"))
